services/web-server/src/servers/createApp.js

```diff
@@ -134,8 +135,19 @@ module.exports = async ({ cfg, strategies, auth, monitor, db }) => {
     getCredentials,
   } = oauth2(cfg, db, strategies, auth, monitor);
 
+  // set up auth rate limiter: maximum # requests per minute
+  const authLimiter = rateLimit({
+    windowMs: 1 * 60 * 1000, // 1 minute
+    max: cfg.app.authRateLimitMaxRequests,
+    legacyHeaders: false, // Disable the `X-RateLimit-*` headers
+  });
+
+  // set number of proxies between the user and the server
+  // to ensure rate limiter doesn't behave like a global one
+  app.set('trust proxy', cfg.app.numberOfProxies);
+
   // 1. Render a dialog asking the user to grant access
-  app.get('/login/oauth/authorize', cors(corsOptions), authorization);
+  app.get('/login/oauth/authorize', authLimiter, cors(corsOptions), authorization);
```

Comment:
so this is a `GET` call which is used to start the authentication process. I don't think it makes sense to rate-limit it at all. 
This is not actively used and is probably invoked by 3rd party (like Treeherder) when they try to authorize using TC as provider